src/AppBundle/Controller/TeamInterestController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Event\TeamInterestCreatedEvent;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use AppBundle\Entity\Department;
+use AppBundle\Entity\TeamInterest;
+use AppBundle\Form\Type\TeamInterestType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+class TeamInterestController extends Controller
+{
+    /**
+     * @Route(name="team_interest_form", path="/teaminteresse/{id}",
+     *     defaults={"id"=null}, requirements={"id"="\d+"})
+     * @Method({"GET","POST"})
+     *
+     * @param \AppBundle\Entity\Department|NULL $department
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
+     */
+    public function showTeamInterestFormAction(Department $department = null, Request $request)
```

Comment:
Ser at det er støtte for å gå til `/teaminteresse` uten å spesifisere department hvis man er logget inn. Vil ikke de som står på stand gå til teaminteressesiden via linken på standsiden i kontrollpanelet (og dermed få med id i url)? Hvis tanken er at man skal slippe å gå gjennom kontrollpanelet, men bare huske at man kan gå til `/teaminteresse`, kan det da bli forvirrende hvis brukeren har glemt å logge inn og får en 400 error?

Reply:
Det var gjort med vilje, men jeg ser nå som du nevner det at det bare blir forvirrende. Stemmer for å bare fjerne `/teaminteresse` pathen, og forlange at department IDen er med.